packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -147,12 +139,11 @@ curl -v -X GET \
 "https://${yourOktaDomain}/api/v1/sessions/101W_juydrDRByB7fUdRyE2JQ"
 ```
 
-##### Response Example
-
+##### Response example
 
-If the session is valid, a [Session object](#session-object) is returned.
+If the Session is valid, a [Session object](#session-object) is returned.
 
-If the session is invalid, a `404 Not Found` response will be returned.
+If the Session is invalid, a `404 Not Found` response will be returned.
```

Comment:
remove future tense

If the Session is invalid, a `404 Not Found` response is returned.